app/controllers/api/databases.php

```diff
@@ -833,7 +838,7 @@ function updateAttribute(
 
         $database = Authorization::skip(fn() => $dbForProject->getDocument('databases', $databaseId));
 
-        if ($database->isEmpty() || (!$database->getAttribute('enabled')  && $mode !== APP_MODE_ADMIN)) {
```

Comment:
why did we remove these checks?

Reply:
These routes are only accessible to API key/admin already so the checks are redundant